Documentation/ColumnsConfig/Type/Select/Properties/ForeignTableWhere.rst

```diff
@@ -13,7 +13,11 @@ foreign_table_where
    :RenderType: all
 
    The items from :ref:`foreign_table <columns-select-properties-foreign-table>`
-   are selected with this WHERE-clause.
+   are selected with this WHERE-clause. The WHERE-clause is effectively
+   appended to the existing WHERE-clause (which contains default constraints,
+   such as 'NOT deleted') and must begin with AND. Additional clauses such as
+   ORDER BY may be added after the WHERE-clause. The result must be a valid SQL
```

Comment:
Which result? This can be misleading as the content of the where clause is not valid SQL.

Maybe something like "The statement to be executed must be valid SQL". But that is obvious.

Reply:
Removed ambiguous sentence.